dmriprep/utils/tests/test_vectors.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import pytest
+import numpy as np
+from pathlib import Path
+from dmriprep.utils import vectors
+
+
+def test_make_and_save_gradient_table():
+    """Test for gradient table generation and saving from bval/bvec."""
+    from dipy.data import fetch_sherbrooke_3shell
+    from dipy.io import read_bvals_bvecs
+    from os.path import expanduser, join
+    from dipy.core.gradients import GradientTable
+
+    fetch_sherbrooke_3shell()
```

Comment:
This will hit dipy servers (whatever CDN they are using) for every test. We can move it to `dmriprep/conftest.py` and run it only once. Deal?

Then you make the files via a fixture (I'm sure you'll love fixtures of pytest).

Reply:
Deal!